### PR TITLE
SB-1406 bugfix: user cache eviction

### PR DESCRIPTION
**Pull request by fuss86**

### CODE REVIEW

File: strongbox-web-core/src/test/resources/etc/conf/strongbox-security-users.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      password: $2a$10$WqtVx7Iio0cndyR1lEaKW.SWhUYmF/zHHG5hkAXvH5hUmklM7QfMO
+      roles:
+        - UI_MANAGER
+        - ARTIFACTS_MANAGER
+      securityTokenKey: deployer-secret
+    - username: admin-cache-eviction-test
+      password: $2a$10$WqtVx7Iio0cndyR1lEaKW.SWhUYmF/zHHG5hkAXvH5hUmklM7QfMO
+      roles:
+      - ADMIN
+      securityTokenKey: admin-cache-eviction-test-secret
```
Comment (steve-todorov) [inline strongbox-web-core/src/test/resources/etc/conf/strongbox-security-users.yaml]: Cosmetic: Missing newline at the end :)

Comment (fuss86) [reply strongbox-web-core/src/test/resources/etc/conf/strongbox-security-users.yaml]: fixed :)

File: strongbox-web-core/src/test/resources/etc/conf/strongbox-security-users.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+users:
+  user:
+    - username: admin
+      password: $2a$10$WqtVx7Iio0cndyR1lEaKW.SWhUYmF/zHHG5hkAXvH5hUmklM7QfMO
+      roles:
+        - ADMIN
```
Comment (carlspring) [inline strongbox-web-core/src/test/resources/etc/conf/strongbox-security-users.yaml]: I believe the formatting of this YAML file is incorrect. The `roles` should be like this:
```
      roles:
      - UI_MANAGER
      - ARTIFACTS_MANAGER
```

Comment (fuss86) [reply strongbox-web-core/src/test/resources/etc/conf/strongbox-security-users.yaml]: It's perfectly valid, see https://symfony.com/doc/current/components/yaml/yaml_format.html

> YAML uses indentation with one or more spaces to describe nested collections:

Also, see our docs https://strongbox.github.io/user-guide/configuration-files/strongbox-security-users-yaml.html

and our source code https://github.com/strongbox/strongbox/blob/master/strongbox-security/strongbox-user-management/src/main/resources/etc/conf/strongbox-security-users.yaml

**Pull request merged**